ansible/group_vars/all.yml

```diff
@@ -1,7 +1,7 @@
 sub_poolid: 8a85f98152a5bd100152a8229dcd3833
-atomic_disk_image: /home/matt.micene/Projects/atomic/vms/rhel/rhel-atomic-cloud-7.2-12.x86_64.qcow2
-rhel_disk_image: /home/matt.micene/Downloads/rhel-guest-image-7.2-20160302.0.x86_64.qcow2
-cfme_disk_image: /home/matt.micene/Downloads/cfme-rhos-5.5.4.2-1.x86_64.qcow2
-kvm_disk_dir: /home/matt.micene/Projects/libvirt-store
-ssh_key: /home/matt.micene/.ssh/id_opentlc.pub
-ssh_known_hosts_file: /home/matt.micene/.ssh/known_hosts
+atomic_disk_image: "{{ lookup('env','HOME') }}/Projects/atomic/vms/rhel/rhel-atomic-cloud-7.2-12.x86_64.qcow2"
+rhel_disk_image: "{{ lookup('env','HOME') }}/Downloads/rhel-guest-image-7.2-20160302.0.x86_64.qcow2"
+cfme_disk_image: "{{ lookup('env','HOME') }}/Downloads/cfme-rhos-5.5.4.2-1.x86_64.qcow2"
+kvm_disk_dir: "{{ lookup('env','HOME') }}/Projects/libvirt-store"
+ssh_key: "{{ lookup('env','HOME') }}/.ssh/id_opentlc.pub"
```

Comment:
This is good stuff, as I was setting up the new laptop I realized this was all hard coded.  Some of these paths are still specific to my old setup or Jamie's (in cfme-demo vars).  Should we pick and document some standard pathing for image locations?

Keeping in mind that all of these can be and should be set up for a particular users workflow.